src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useContext } from 'react';
+import { PeopleContext } from '../../Context';
+import classNames from 'classnames';
+import { Link, useLocation } from 'react-router-dom';
+import { SortValues } from '../../enums/SortValues';
+
+export const PeopleTable: React.FC = () => {
+  const {
+    searchParams,
+    people,
+    filteredPeople,
+    sortParam,
+    orderParam,
+    sortHandler,
+  } = useContext(PeopleContext);
+  const { pathname } = useLocation();
+
+  // #region functions
+
+  const capitalize = (word: string) =>
+    word.charAt(0).toUpperCase() + word.slice(1);
+
+  const sexColor = (sex: string) => {
+    return classNames({ 'has-text-danger': sex === 'f' });
+  };
+
+  // #endregion
+  // #region getParents
+
+  const getMother = (_motherName: string | null) => {
+    const mother = people.find(_person => _motherName === _person.name);
```

Comment:
The variable `_motherName` is used but not defined in the `getMother` function. Ensure that `_motherName` is passed correctly as a parameter to avoid potential errors.